client/lib/media/actions.js

```diff
@@ -260,7 +260,9 @@ MediaActions.update = function( siteId, item, editMediaFile = false ) {
 				type: 'RECEIVE_MEDIA_ITEM',
 				error: error,
 				siteId: siteId,
-				data: data
+				data: editMediaFile
+					? { ...data, isDirty: true }
```

Comment:
That's true, it's just much more confusing and right now I have no idea if nothing happens..